src/core/style/mixins.scss

```diff
@@ -114,7 +114,12 @@
 @mixin rtl-prop($ltr-prop, $rtl-prop, $value) {
   #{$ltr-prop}: $value;
   [dir=rtl] & {
+    // fallback for auto
+    #{$ltr-prop}: 0;
+    // fallback for IE
     #{$ltr-prop}: auto;
+    #{$ltr-prop}: initial;
```

Comment:
This is by intention,
What we really want: `initial`
What we feel meh about: `auto`
What we settle for: `0`